### ISSUE TITLE
qweechat: depends on python27Packages.setuptools to run

### ISSUE THREAD

**Issue by bqv** [NixOS/nixpkgs#84253]
To reproduce:
```bash
$ nix-shell -p qweechat --run qweechat --pure
Traceback (most recent call last):
  File "/nix/store/jvqd77r7ppl1dw5kn29hfp3wzl2y95l7-qweechat-unstable-2016-07-29/bin/.qweechat-wrapped", line 6, in <module>
    from qweechat.qweechat import main
  File "/nix/store/jvqd77r7ppl1dw5kn29hfp3wzl2y95l7-qweechat-unstable-2016-07-29/lib/python2.7/site-packages/qweechat/qweechat.py", line 38, in <module>
    from pkg_resources import resource_filename
ImportError: No module named pkg_resources
```

Workaround:
`nix-shell -p python27Packages.setuptools qweechat --run qweechat --pure`

**Comment by bqv** [comment]
Did it myself. #84264

**Comment by bhipple** [comment]
As a tip, in the PR body you can say something like `Closes #84253` or `Resolves #84253`, and GitHub will automatically mark this issue as closed when the PR is merged.

**Comment by bqv** [comment]
Gotcha
